packages/fast-components-react-base/src/label/label.spec.tsx

```diff
@@ -79,4 +79,16 @@ describe("label", (): void => {
         expect(rendered.instance().props.tag).toEqual(LabelTag.legend);
         expect(rendered.type()).toBe(LabelTag.legend);
     });
+
+    test("should rendered the correct className base on hidden prop", () => {

```

Comment:
this test naming structure, although we are doing something similar above, is wrong, tests should always pass therefore the word "correct" is redundant, I would rather see two tests, one when the hidden prop is not passed with the description "should add the base className" and when the hidden prop is passed "should add the hidden className and the base className when the hidden prop is passed"

Reply:
updated